app/controllers/trips_controller.rb

```diff
@@ -0,0 +1,28 @@
+class TripsController < ApplicationController
+skip_before_action :authenticate_user!, only: [ :create, :show ]
+before_action :set_trip, only: [ :show ]
+
+  def create
+    @trip = Trip.new(trip_params)
+    # assign user_id to trips if user is logged in
+    @trip.user = current_user
+    if @trip.save
+      redirect_to trip_path(@trip)
+    else
+      render 'pages/home'
```

Comment:
Shouldn't it redirect to the path for trips#show?

Reply:
exactly. this is the path for trip show. check routes!!!!